README.md

```diff
@@ -1,4 +1,29 @@
-# 2439challenge1
-a template repository for the 2439 midterm project
+# Applied Challenge 1 - Carter Brehm
 
-[Applied Challenge #1 Instructions](https://docs.google.com/document/d/e/2PACX-1vTbclFb83xohS9hWkVu7YA7JnSVJqh5Lz53HwQE6UONAhgtpvLjYeUGF1lqurwyjg/pub)
+
+## Scope Statement
+The application will allow a user to learn about the restaurant before visiting, and submit feedback about their experience after their visit. 
+
+The user will be able to view the menu of the restaurant sorted by category (including specials), view the hours and location of the restaurant (including special holiday hours), and make a reservation using OpenTable embedded on the site. Then, when the user has eaten at the restaurant, they can take a survey on which they can provide feedback about the food, service, and general experience.
+
+## Mockup / Wireframe
+
+[Moqup](https://app.moqups.com/LSmgBmwpix/view/page/aa9df7b72?ui=0&fit_width=1)
```

Comment:
You have a nice start here! I would have loved to see you make use of the phone templates so that we could see how the layout will ideally adapt to the smaller screen.